components/PublicationDate/src/stories/publicationDate.bem.stories.mdx

```diff
@@ -0,0 +1,37 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
```

Comment:
This file should probably be called `bem.stories.mdx`. We don't use the component name in the stories files anymore.

Reply:
@bddjong I renamed the story file in commit: https://github.com/nl-design-system/denhaag/pull/662/commits/ed93c2636a22338d5b7acaf1c5e49d3d4e01f4c3